lua/lint/linters/luacheck.lua

```diff
@@ -0,0 +1,40 @@
+local severities = {
+    W = vim.lsp.protocol.DiagnosticSeverity.Warning,
+    E = vim.lsp.protocol.DiagnosticSeverity.Error
+}
+
+local pattern = '[^:]+:(%d+):(%d+)-(%d+): %((%a)(%d+)%) (.*)'
+
+return {
+    cmd = 'luacheck',
+    stdin = true,
+    args = {'--formatter', 'plain', '--codes', '--ranges', '-'},
```

Comment:
I think this also needs a `ignore_exitcode = false,` to suppress the "Linter command luacheck exited with code 1" messages.

Reply:
Shouldn't it be `ignore_exitcode = true`? I'm still getting the messages with `false` but they're gone with `true`.

Reply:
Right, it should be `true`

Reply:
Fixed.